helix-view/src/editor.rs

```diff
@@ -592,6 +594,51 @@ impl Default for IndentGuidesConfig {
     }
 }
 
+/// Line ending configuration.
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "lowercase")]
+pub enum LineEndingConfig {
+    /// The platform's native line ending.
+    ///
+    /// `crlf` on Windows, otherwise `lf`.
+    Native,
```

Comment:
I found `native` to be the clearest here, but I'm open to alternatives (`platform`, `system`, `environment`, etc.)